cogs/starboard.py

```diff
@@ -0,0 +1,128 @@
+from typing import List, Optional
+import discord
+from discord.ext import commands
+from discord import ui
+
+STARBOARD_CHANNEL_ID = 1133260871049691257  # 1139734225390669875
+ON_PHONE_EMOJI_STR = "<:OnPhone:1062142401973588039>"  # "<:OnP:1139749947357544500>"
+
+
+class Starboard(commands.Cog):
+    def __init__(self, bot: discord.Client):
+        self.starboard_emoji_str = ON_PHONE_EMOJI_STR
+        self.threshold = 1
+        # Starboarded Message ID -> ID of the message the bot sent.
+        self.starboard_msgs = dict()
+        self.starboard_channel = bot.get_channel(STARBOARD_CHANNEL_ID)
+
+    async def cog_load(self):
+        await super().cog_load()
+        print("Starboard Cog loaded.")
+
+    def _get_title(self, react: discord.Reaction) -> str:
```

Comment:
if the starboarded message is a reply it's more clear to the reader if the title is "<starboarded user> replied:".